spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/authentication/RSAProviderTokenManager.java

```diff
@@ -21,33 +21,41 @@
 import java.util.Map;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.core.env.Environment;
 
 public class RSAProviderTokenManager {
   private static final Logger LOGGER = LoggerFactory.getLogger(RSAProviderTokenManager.class);
 
   private final List<AccessController> accessControllers;
 
-  public RSAProviderTokenManager(List<AccessController> accessControllers) {
+  private final Environment environment;
+
+  public RSAProviderTokenManager(List<AccessController> accessControllers, Environment environment) {
     this.accessControllers = accessControllers;
+    this.environment = environment;
   }
 
   public void valid(String token, Map<String, String> requestMap) throws Exception {
     try {
-      boolean isAllow = true;
+      RsaAuthenticationToken rsaToken = null;
+      if (environment.getProperty(Const.AUTH_TOKEN_CHECK_ENABLED, Boolean.class, true)) {
+        rsaToken = RSATokenCheckUtils.checkTokenInfo(token);
```

Comment:
Hide `RSATokenCheckUtils.validatedToken.put(rsaToken, true);` in  `RSATokenCheckUtils.checkTokenInfo(token)`

Reply:
fixed